### DISCUSSION TITLE
What can orchard core do?

### DISCUSSION THREAD

**Discussion by uussoft** [OrchardCMS/OrchardCore#7251] [Q&A]
What can orchard core do?

**Comment by hishamco** [comment]
What Orchard Core couldn't do for you while you developing a website or web application ;)

OC is a web framework and CMS on top of that framework, there 're more than 50 modules out of the box that let you as a web developer do whatever you want. Also there are extensibility points that allows you to replace existing components or modules if didn't fit your needs

**Reply by uussoft** [reply]
thanks！

**Reply by jtkech** [reply]
Many things, e.g. a web app hosting a site or multi sites, and with modules / features that can be enabled / disabled per site ;)

**Comment by agriffard** [comment]
Currently trying to refactor the list of features in the doc: https://github.com/OrchardCMS/OrchardCore/pull/6942
